CHANGELOG.md

```diff
@@ -4,6 +4,31 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## Unreleased
+
+**Bugfixes**
+* #4619 Improve handling enum columns in compressed hypertables
+* #4673 Fix now() constification for VIEWs
+* #4676 Fix a deadlock when decompressing chunks and performing SELECTs
+* #4681 Fix compression_chunk_size primary key
+* #4685 Improve chunk exclusion for space dimensions
+* #4696 Report warning when enabling compression on hypertable
+* #4720 Fix chunk exclusion for prepared statements and dst changes
+* #4737 Fix join on time column of compressed chunk
+* #4738 Fix the assorted epoll_ctl() errors that could occur with COPY into a distributed hypertable
+* #4739 Fix continuous aggregate migrate check constraint
+* #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
+* #4756 Improve compression job IO performance
+* #4760 Fix segfault when INNER JOINing hypertables
+* #4735 Allow manual index creation for CAggs
```

Comment:
Same as pointed out in the other release issue: https://github.com/timescale/timescaledb/pull/4773#discussion_r984809457

Reply:
@fabriziomello Thank you for the hint. It is fixed in the current version of the PR.